src/backend/controllers/battleController.js

```diff
@@ -205,7 +206,7 @@ exports.prepareMatch3v3 = async (req: Request, res: Response) => {
 				.status(404)
 				.json({ msg: 'Could not find the challenger in DB'});
 		}
-		// Make sure they have enuf for the wager
+		// Make sure they have enough for the wager
```

Comment:
lets be real the spelling of enough should be enuf

Reply:
lmfao why Baylor